packages/css/src/dso.scss

```diff
@@ -1,105 +1,91 @@
-// Import variables
-@import "../../sources/src/styling/variables";
+@use "@dso-toolkit/sources/src/utilities" as utilities-mixins;
```

Comment:
Stond hier nou eerst `as shared-utilities`? Mijn vorige opmerking ging erover dat we gewoon utilities consumeren als `utilities.box-sizing();` :D

Reply:
Dit komt vanwege conflicterende namespacing, laten we zo